docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -179,25 +183,44 @@ generate the certificate specified by `--fleet-server-cert`.
 [role="screenshot"]
 image::images/add-fleet-server-advanced.png[In-product instructions for adding a {fleet-server} in advanced mode]
 
-. Then, use the in-product instructions to install the {agent}.
+. Step through the in-product instructions to configure and install {fleet-server}.
 +
 [NOTE]
 ====
-The `install` command installs the {agent} as a managed service and enrolls it
-in a {fleet-server} policy. For more {fleet-server} commands, see
-{fleet-guide}/elastic-agent-cmd-options.html[{agent} command reference].
+* The fields to configure {fleet-server} hosts are not available if the hosts
+are already configured outside of {fleet}. For more information, refer to 
+{kibana-ref}/fleet-settings-kb.html[{fleet} settings in {kib}].
+* When using the *Advanced* option, it's recommended to generate a unique service
+token for each {fleet-server}. For other ways to generate service tokens, refer to
+{ref}/service-tokens-command.html[`elasticsearch-service-tokens`].
+* If you've configured a non-default port for {fleet-server} in the 
+{fleet-server} integration, you need to include the `--fleet-server-host` and 
+`--fleet-server-port` options in the `elastic-agent install` command. Refer to the 
+{fleet-guide}/elastic-agent-cmd-options.html#elastic-agent-install-command[install command documentation]
+for details.
```

Comment:
@lucabelluccini, here's the note that I added to address [your comment](https://github.com/elastic/ingest-docs/issues/99#issuecomment-1506176447) in the parent issue. Please let me know if I didn't get it right. :-)

![Screenshot 2023-05-16 at 6 22 09 PM](https://github.com/elastic/ingest-docs/assets/41695641/783007ec-f503-4f32-9274-50eea925ef44)